app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -333,7 +344,7 @@ def update
     else
       return render status: :unprocessable_entity, json: { error: "Invalid group type" }
     end
-    RoleChangeMailer.notify_role_change(role, current_user, changed_parameter, previous_value, new_value).deliver_later
+    RoleChangeMailer.notify_role_change(role, current_user, changes).deliver_now # Change to deliver_later
```

Comment:
There's a comment at the end of the line which feels relevant

Reply:
Oops sorry I missed to remove it after testing 😅 Removed now.